docs/Examples/Basic.md

```diff
@@ -140,20 +144,15 @@ public func url(route: TargetType) -> String {
 }
 
 let endpointClosure = { (target: MyService) -> Endpoint<MyService> in
-    return Endpoint<MyService>(url: url(target), sampleResponseClosure: {.networkResponse(200, target.sampleData)}, method: target.method, parameters: target.parameters)
+    return Endpoint<MyService>(url: url(target), sampleResponseClosure: {.networkResponse(200, target.sampleData)}, method: target.method, task: target.task)
 }
 ```
 
 The block you provide will be invoked every time an API call is to be made. Its
 responsibility is to return an `Endpoint` instance configured for use by Moya.
-`parameters` is passed into this block to allow you to configure the `Endpoint`
```

Comment:
I did not really understand this sentence and I'm also not sure if it's needed with the new `task` type. So I removed it. I would need an explanation if this should be kept and updated.